frontend/admin-client/src/app/bookings/booking-list/booking-list.component.html

```diff
@@ -65,18 +66,13 @@
       </div>
     </div>
     <div class="d-flex justify-content-center" style="margin: 150px">
-      <app-pagination *ngIf="pager.totalPages && pager.currentPage"
-        [collectionSize]="pager.totalPages"
-        [maxSize]="5"
-        [(page)]="pager.currentPage"
-        (pageChange)="navigate($event)"
-
-      ></app-pagination>
+      <app-pagination *ngIf="pager.totalPages && pager.currentPage" [(currentPage)]="pager.currentPage"
```

Comment:
I see visiblePageCount is set to 5  here, but it shows 10 when I run it.

Reply:
Change it to 2. For simplicity, I made it so that it is the number of pages on either side.